guide/installation-and-configuration/pre-installation-checklist/verify-signatures.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Verifying package signatures
+published: true
+sorting: 40
+tags: [getting started, installation]
+---
+
+On the [Download CFEngine][enterprise software download page], you will find
+sha256 checksums of all downloadable files which you can verify by using
+`sha256sum` tool.
+
+In addition to this, `*.deb` and `*.rpm` packages (with except of AIX rpm) are
+cryptographically signed using gpg.
+
+
+## Validating signature of RPM
+
+NOTE: The AIX rpm is currently NOT signed because it's not supported on older versions of AIX.
```

Comment:
MMMm we only make one version of the AIX rpm huh? I guess we could start publishing different versions like we do for various versons of redhat or debian.

Reply:
we publish both aix 6 and aix 7, indeed